sd-step.go

```diff
@@ -86,10 +93,49 @@ func execHab(pkgName string, pkgVersion string, command []string, output io.Writ
 	return nil
 }
 
+// getPackageVersion returns the appropriate package version which matched the `pkgVerExp` expression.
+func getPackageVersion(depot hab.Depot, pkgName, pkgVerExp string) (string, error) {
+	versionConst, err := semver.NewConstraint(pkgVerExp)
+	// if pkgVerExp is invalid for semver expression, it returns pkgVerExp as it is
+	if err != nil {
+		return pkgVerExp, nil
+	}
+
+	foundVersions, err := depot.PackageVersionsFromName(pkgName)
+	if err != nil {
+		return "", errors.New(fmt.Sprintf("Failed to fetch package versions: %v", err))
```

Comment:
`golint` says `sd-step.go:106:14: should replace errors.New(fmt.Sprintf(...)) with fmt.Errorf(...)`.
So we should use fmt.Errorf() instead of errors.New().